### ISSUE TITLE
Icons visual information not communicated in text (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-progress-indicator#43]
### Issue from RE NXT A11y audit 2019
**Avoid use of the before, after, pseudo elements for non-decorative content**
When the before, after pseudo elements are used to display non-decorative content users of assistive technology will likely not have access to this text or image information. Without access to this information the content of the page may be incorrectly interpreted.

CSS before and after pseudo-elements are used to indicate Progress indicator steps.

### Impact
No textual equivalents are available for the progress indicator steps as CSS pseudo-elements are not rendered to screen reader users.

This impacts all uses of progress indicator - Passive, Waterfall, Wizard
